src/windows/settings_utils.cpp

```diff
@@ -300,7 +300,7 @@ namespace display_device::win_utils {
         // It's hard to deal with floating values, so we just multiply it
         // to keep 4 decimal places (if any) and let Windows deal with it!
         // Genius idea if I'm being honest.
-        constexpr auto multiplier { static_cast<unsigned int>(std::pow(10, 4)) };
+        static const auto multiplier { static_cast<unsigned int>(std::pow(10, 4)) };
```

Comment:
Maybe just write this as 10000?
Without floating point math, integer casts, static variables and fighting compilers.